help/en/html/FAQ.shtml

```diff
@@ -205,17 +205,17 @@
         MORE
         </dd>
  -->       
-        <dt id="develops">Who develops and maintains JMRI?</dt>
+        <dt id="develops"><strong>Who develops and maintains JMRI?</strong></dt>
 
         <dd>
-        We do!  JMRI is open source and community maintained.  There are over 300 model railroaders who ahve already made
+        We do!  JMRI is open source and community maintained.  There are over 300 model railroaders who have already made
         contributions to the effort since 2005 (see their names on the 
         <a href="../Acknowledgements.shtml">Acknowledgements</a> page). Anyone with knowledge of Java
-        (to maintain the programs), HTML (to maintain the programs), or XML (to maintain decoder definitions) can join in. YOu can
-        also become one of the almost 10,000 registered with the JMRI Users forum on groups.io to ask and answer questions.  
+        (to maintain the programs), HTML (to maintain the documentation), or XML (to maintain decoder definitions) can join in. Encourage you to
```

Comment:
Line 214:  word missing before "Encourage you"  - perhaps leave as is?

Reply:
Adjusted the line